src/main/webapp/src/app/recommendation/recommendation-item/recommendation-item.component.ts

```diff
@@ -1,13 +1,30 @@
-import { Component, Input } from '@angular/core';
-import { HiddenGem } from 'src/app/hidden-gem';
+import { Component, Input, OnInit } from '@angular/core';
+import { HiddenGem } from '../../models/hidden-gem';
 
 
 @Component({
   selector: 'app-recommendation-item',
   templateUrl: './recommendation-item.component.html',
   styleUrls: ['./recommendation-item.component.css']
 })
-export class RecommendationItemComponent {
+export class RecommendationItemComponent implements OnInit{
   @Input() hiddenGem = {} as HiddenGem;
+  photoUrl!: string;
+  type!: string;
 
+  ngOnInit() {
+    if (this.hiddenGem.photos != null) {
+      this.photoUrl = "https://maps.googleapis.com/maps/api/place/photo?maxwidth=400&photoreference="
+        + this.hiddenGem.photos[0].photoReference
+        + "&key=AIzaSyCBb8QQBQal9jDNl3ZG6f3bS6ROX2MtYIM";
```

Comment:
Is this a restricted key?

Reply:
Yes, for now I used the same key as for the Maps JavaScript API